src/main/java/com/demonwav/mcdev/buildsystem/gradle/GradleBuildSystem.java

```diff
@@ -215,7 +215,7 @@ private void setupWrapper(@NotNull Project project, @NotNull ProgressIndicator i
             try {
                 String wrapperDirPath = getRootDirectory().createChildDirectory(this, "gradle").createChildDirectory(this, "wrapper").getPath();
                 FileUtils.writeLines(new File(wrapperDirPath, "gradle-wrapper.properties"), Collections.singletonList(
-                    "distributionUrl=https\\://services.gradle.org/distributions/gradle-2.14.1-bin.zip"
+                    "distributionUrl=https\\://services.gradle.org/distributions/gradle-3.4.1-bin.zip"
```

Comment:
3.4.1 isn't a good idea because it will cause problems when using it with IntelliJ IDEA 2016.3. Gradle 3.3 is better for now.